src/main/kotlin/models/ImageModel.kt

```diff
@@ -0,0 +1,7 @@
+package models
+
+import tornadofx.*
+
+class ImageModel(image: IPEwGImage): ItemViewModel<IPEwGImage>(image) {
```

Comment:
Needs to add a space on the left of `:`  for code formatting (not trying to be picky... just to make sure)